services/user.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str | None = None,
+        first_name: str | None = None,
+        last_name: str | None = None,
+) -> None:
+    user = User(username=username)
+    user.first_name = first_name if first_name else ""
+    user.email = email if email else ""
+    user.last_name = last_name if last_name else ""
+
+    with transaction.atomic():
+        user.set_password(password)
+        user.save()
```

Comment:
do you think only password can cause an error?
What if you get incorrect email like someemail.com?
Or the length of first_name will be > than max_length of this field?